README.md

```diff
@@ -45,6 +45,67 @@ you can enable/disable for the command-line app, use the `--help` option:
 addons-linter --help
 ```
 
+### Import Linter API into another NodeJS application
+
+```js
+import {createInstance as createLinter} from 'addons-linter';
+
+async lint(sourceDir) {
+  const linter = createLinter({
+    config: {
+      // This mimics the first command line argument from yargs,
+      // which should be the directory to the extension.
+      _: [sourceDir],
+      logLevel: process.env.VERBOSE ? 'debug' : 'fatal',
+      stack: Boolean(process.env.VERBOSE),
+      pretty: false,
+      warningsAsErrors: false,
+      metadata: false,
+      output: 'json',
+      boring: false,
+      selfHosted: false,
+      // Lint only the selected files
+      //   scanFile: ['path/...', ...]
+      //
+      // Exclude files:
+      shouldScanFile: (fileName) => true,
+    },
+    // This prevent the linter to exit the nodejs application
+    runAsBinary: false,
+  });
+
+  linter.print = () => {}; // Prevents the linter from printing the report
```

Comment:
yeah, I agree: I tweaked both the example and the Linter class so that a config with 'type: "none"' does not produce any linter output.